src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+// ACTIONS
+const ADD = 'my-app/widgets/ADD';
+const REMOVE = 'my-app/widgets/REMOVE';
+
+export default function booksReducer(state = {}, action = {}) {
```

Comment:
- [ ] The initial state of your global store should be an empty array literal (`[ ]`) and not an empty object literal (`{ }`). Kindly follow the example of the first reviewer. 👍 

![image](https://user-images.githubusercontent.com/77106962/207161416-351fc513-6e22-4237-8c92-3cccdf6a0765.png)